pkg/ddevapp/envfile.go

```diff
@@ -25,9 +26,9 @@ func WriteProjectEnvFile(envFilePath string, envMap map[string]string, envText s
 	for k, v := range envMap {
 		// If the item is already in envText, use regex to replace it
 		// otherwise, append it to the envText.
-		exp := regexp.MustCompile(fmt.Sprintf(`((^|[\r\n]+)%s)=(.*)`, k))
+		exp := regexp.MustCompile(fmt.Sprintf(`(^|[\r\n]+)#*\s*(%s)=(.*)`, k))
```

Comment:
Please add a comment explaining the purpose of this regex. You apparently had to study it!

Reply:
Done.